.github/workflows/test.yml

```
@@ -37,6 +37,11 @@ jobs:
       - name: Run cargo doc
         if: always()
         run: cargo doc --workspace --bins
+      - name: Test build without llvm feature
+        if: always()
```

Can this wait until the "normal" build passes? I feel like this is very rarely of immediate concern for PR authors, it'll most likely fail too if the build fails and it wouldn't help anyways if this succeeds but the "normal" build fails. 

```suggestion
        if: always()
        needs: linux-x86-64
```

---

I don't think you can make a step depend on another job, I think we need a whole new job that depends on linux-x86-64. wdyt?

---

Right, the `needs` should go to the job but I couldn't suggest it there. But now that I think about it, shouldn't this step be inside `linux-x86-64` anyways?

---

I've changed it again. Now I use `cargo clippy --no-default-features --lib -- -D warnings`. This gives build errors, warnings, and clippies for building without the llvm feature. Also, hopefully you'll agree this fits into the lints job better.

---

This is even better